code/script.js

```diff
@@ -1,14 +1,22 @@
 // Variables that point to selected DOM elements
 const chat = document.getElementById('chat');
+const form = document.getElementById('name-form');
+const inputWrapper = document.getElementById('input-wrapper');
+const sendBtn = document.getElementById('send-btn');
 
-// If you need any global variables that you can use across different functions, declare them here:
+const botReply = (msg) => {
+  showMessage(msg, 'bot')
+};
 
+const userReply = (msg) => {
+  showMessage(msg, 'user')
+};
```

Comment:
Just a thought! You could include the setTimeout in botReply and userReply functions, if you want to simplify the code further below and if you use the same timer for all responses.